### ISSUE TITLE
Inaccessible page: Noise Margin

### ISSUE THREAD

**Issue by supragya** [Sunscreen-tech/Sunscreen#257]
The deployed version of docs.sunscreen.tech on page https://docs.sunscreen.tech/fhe_programs/writing_an_fhe_program/limitations.html links to 
https://docs.sunscreen.tech/fhe_programs/writing_an_fhe_program/advanced/noise_margin.html under "Bounded Computation". 

Publically, this translates to the following view on webpage:
```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>WS4B9GEZTBEF4BPF</RequestId>
<HostId>f9+abKGPtQPCSw/hLVsjYM2GbmBGtX8ZO1hDeNwU6v4ELd4OiIGwyTXKK83jPpLNzfqPgKVdsY0=</HostId>
</Error>
```

**Comment by supragya** [comment]
Similar report for https://docs.sunscreen.tech/fhe_programs/plaintext_execution.html

**Comment by supragya** [comment]
Similar report for https://docs.sunscreen.tech/advanced/batching/batching.html

**Comment by samtay** [comment]
Thanks for letting us now! I'll fix these up and get new docs deployed shortly :+1:

**Comment by samtay** [comment]
Closed by #259
